File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.cc
#include <stdio.h>
int main(){
    int x, y;
    int count;
    scanf("%d\n", &count);
    for(int i = 1 ; i != count + 1; i++){
        scanf("%d ", &x);
        scanf("%d\n", &y);
        printf("Case #%d: %d + %d = %d\n", i, x, y, x + y);
        }
    }

File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int x,y;
        int count = sc.nextInt() + 1; 
        for(int i = 1 ; i != count; i++){
            x = sc.nextInt();
            y = sc.nextInt();
            System.out.printf("Case #%d: %d + %d = %d\n", i, x, y, x + y);
        }
        
    } 
}